src/Illuminate/Database/Connection.php

```diff
@@ -27,21 +28,28 @@ class Connection implements ConnectionInterface
      *
      * @var PDO
      */
-    protected $pdo;
+    private $pdo;
```

Comment:
Changed to private in favor of getter/setters to handle creating connections when called for the first time. See the `getPdo()` and `getReadPdo()` methods